snapcraft/cli/store.py

```diff
@@ -295,11 +301,15 @@ def export_login(login_file: TextIO, snaps: str, channels: str, acls: str):
     For example, to limit access to the edge channel of any snap the account
     can access:
 
-        snapcraft export-login --channels=edge
+        snapcraft export-login --channels=edge exported
 
     Or to limit access to only the edge channel of a single snap:
 
-        snapcraft export-login --snaps=my-snap --channels=edge
+        snapcraft export-login --snaps=my-snap --channels=edge exported
+
+    To limit access to a single snap, but only until the first of the year:
```

Comment:
first *day*.

I think it would be clearer to use the last day of the year in the example, but that's just a nit.

Reply:
Well, this is midnight of the first day of the new year, which makes sense, right? I did reword to say "only until 2019" and then updated it to use midnight of the first day of 2019.